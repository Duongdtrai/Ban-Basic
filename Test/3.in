20
79
44 22 5 59 75 1 9 17 34 47 30 63 89 37 92 66 69 63 28 28 7 95 82 98 6 42 50 59 65 87 21 33 79 15 66 31 65 40 81 50 51 14 54 98 40 2 61 21 73 43 42 21 81 91 68 3 48 56 92 60 37 2 24 16 85 57 24 29 52 37 85 54 63 6 59 40 39 8 53 
96
22 45 1 58 79 95 100 11 70 73 17 74 93 60 35 48 63 34 79 9 94 74 42 60 15 38 7 63 51 16 55 2 85 100 43 5 97 19 39 71 51 98 87 75 26 53 4 66 93 52 7 90 92 72 63 84 76 37 65 59 77 84 88 90 48 3 88 66 27 40 72 41 83 53 28 6 64 6 56 12 69 43 62 64 5 50 70 70 46 6 15 31 33 47 77 16 
36
68 51 77 47 3 90 30 71 23 98 54 93 34 99 43 65 96 97 61 41 95 91 8 98 98 56 21 16 83 75 45 43 81 98 46 91 
76
14 5 31 54 87 41 65 8 64 100 19 96 23 4 13 94 76 71 24 96 41 48 22 25 85 43 95 76 16 24 7 45 59 28 55 44 45 50 55 77 73 91 9 49 89 22 21 41 39 84 97 16 37 29 10 61 53 31 97 71 21 41 10 33 79 48 88 43 35 23 93 93 99 12 2 40 
36
84 20 88 10 25 47 62 68 2 2 17 57 53 98 34 30 30 74 7 35 36 48 98 13 7 23 42 13 34 44 48 50 60 71 67 58 
90
2 27 12 8 71 42 64 28 15 34 67 21 75 65 23 29 50 6 19 89 42 4 39 79 8 33 59 92 1 41 25 63 23 94 100 96 18 37 9 100 28 46 79 37 94 95 12 55 19 31 1 9 55 22 9 48 86 61 34 89 21 17 49 54 86 16 95 41 26 54 63 40 16 12 59 24 54 53 5 78 17 91 17 10 93 34 24 77 95 44 
72
85 99 8 73 55 50 77 97 75 75 37 8 69 9 62 28 34 84 80 22 30 7 5 76 62 25 76 96 69 5 70 1 27 59 68 81 69 27 67 69 14 43 99 44 95 52 40 66 70 12 50 64 36 74 21 16 33 70 41 64 81 70 41 62 99 69 29 28 6 59 19 55 
28
77 86 45 22 41 31 94 72 3 32 34 67 29 8 61 59 60 38 26 72 30 53 95 30 97 47 70 11 
97
8 19 86 32 91 14 77 29 13 71 96 8 49 98 88 57 14 99 77 24 32 7 66 16 43 93 27 21 44 47 93 41 11 40 72 21 91 87 66 87 52 23 20 81 82 29 17 81 98 25 89 4 49 99 86 46 61 32 33 26 4 81 100 45 6 50 61 67 50 33 69 8 53 12 51 5 11 25 54 69 84 19 19 41 40 2 45 91 32 25 51 22 79 96 40 12 18 
23
83 49 73 84 96 8 42 50 18 3 38 48 90 83 55 97 44 78 90 98 68 87 27 
23
17 25 20 41 16 69 90 40 46 79 72 85 32 53 62 15 59 24 38 46 13 73 52 
100
89 19 53 20 70 16 22 88 57 99 40 60 27 73 77 98 19 34 66 34 80 68 58 43 61 63 16 15 5 94 47 14 39 47 42 11 8 55 30 67 34 90 6 92 51 72 63 54 74 40 64 93 49 31 10 92 6 91 41 9 2 56 57 52 75 73 4 34 43 54 22 70 47 74 91 6 89 76 100 25 66 14 62 73 87 89 91 4 78 34 61 76 82 79 71 13 59 54 26 27 
25
30 79 3 1 6 29 95 49 92 72 50 55 85 25 28 83 31 44 40 36 100 51 22 70 66 
3
18 16 43 
22
51 98 13 70 52 100 12 46 8 50 16 51 72 27 67 83 94 32 29 24 70 14 
45
36 69 71 74 72 68 78 79 10 63 57 30 60 47 7 83 51 84 18 89 87 38 76 6 7 72 42 97 26 89 35 10 96 5 80 97 49 66 4 43 57 27 65 35 68 
20
74 28 45 73 55 9 2 67 39 83 99 48 80 68 70 69 58 25 14 85 
6
33 23 20 87 54 30 
44
87 97 56 59 38 57 62 17 94 43 21 16 86 53 23 70 4 35 44 67 3 81 25 91 43 26 26 13 94 42 34 37 81 19 69 82 26 77 74 99 9 63 65 82 
19
43 45 43 38 87 96 59 67 5 76 99 56 65 39 33 95 77 35 60 
