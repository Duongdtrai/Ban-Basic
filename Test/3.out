5071866676
2668860780
207220422
4937241986
192331830
1065943696
4168205340
43207050
5708115096
65382030
9246592
17215346148
1185948
688
28968498
656869938
28440280
55552
305247748
26510820
